### PR TITLE
Rate limit clarifications

### PR DESCRIPTION
**Pull request by parautenbach**
## Proposed change

Clarify rate limits for certain templating functions.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/configuration/templating.markdown [hunk 1]
```diff
@@ -1296,3 +1296,31 @@ The default priority of operators is that the filter (`|`) has priority over eve
 {% endraw %}
 
 Would round `10` to 2 decimal places, then divide `states('sensor.temperature')` by `10` (rounded to 2 decimal places so 10.00). This behavior is maybe not the one expected, but priority rules imply that.
+
+### Rate limits for template updates
```
Comment (ludeeus) [inline source/_docs/configuration/templating.markdown:1300]: This section seems out of place, this document introduces all the different functions.
None of these are rate-limited as-is. However, some rate-limiting happens in the [automation trigger](https://www.home-assistant.io/docs/automation/trigger/#template-trigger) and [template entity](https://www.home-assistant.io/integrations/template/) updates. Did you mean to add this to those places instead?

**Comment by frenck** [comment]
I've marked this PR, as changes are requested that need to be processed.
Please un-draft it once it is ready for review again by clicking the **"Ready for review"** button.

Thanks! 👍 

../Frenck

[_Learn more about our pull request process._](https://developers.home-assistant.io/docs/review-process#prs-are-being-drafted-when-changes-are-needed)

**Comment by frenck** [comment]
Since there was no response at all, I'm declaring this PR stale and closing it.
Feel free to re-open this PR when you feel like that is the appropriate thing to do.

File: source/_docs/configuration/templating.markdown (hunk 1 above)
Comment (parautenbach) [reply source/_docs/configuration/templating.markdown:1300]: I wasn't sure where the best place was, to be honest. I'll be happy to add it there. Also, I think there would be some duplication of information in that case. Would that be a problem?

**Comment by parautenbach** [comment]
Apologies, Frenck. I somehow completely missed the feedback I received here. 

You can keep it closed though. I'm following up on Ludeeus' comment.

File: source/_docs/configuration/templating.markdown (hunk 1 above)
Comment (ludeeus) [reply source/_docs/configuration/templating.markdown:1300]: They are two different things, so I do not think that will be an issue.

**Pull request closed without merging**